pkg/kubelet/userns/userns_manager_test.go

```diff
@@ -345,6 +348,10 @@ func TestGetOrCreateUserNamespaceMappings(t *testing.T) {
 
 	for _, tc := range cases {
 		t.Run(tc.name, func(t *testing.T) {
+			if tc.skipOnWindows && goruntime.GOOS == "windows" {
+				// TODO: remove skip once the failing test has been fixed.
+				t.Skip("Skip failing test on Windows.")
+			}
```

Comment:
This is the wrong fix for user namespaces. It will be GREAT if I can be cc for userns changes, as this is causing other issues now :(